frontend/src/Editor/Components/Table/Table.jsx

```diff
@@ -276,10 +276,17 @@ export function Table({
         const cellValue = rowChangeSet ? rowChangeSet[column.name] || cell.value : cell.value;
 
         if (columnType === 'string' || columnType === undefined || columnType === 'default') {
+          
+          const textColor = resolveReferences(column.textColor, currentState, { cellValue }) || [];
```

Comment:
Probably have a fallback default color if value is empty?

```suggestion
          const textColor = resolveReferences(column.textColor, currentState, { cellValue }) || '#000';
```

Reply:
Right, we also need to consider '#fff' as the default if dark mode.

Reply:
It's much better if you create some predefined color constants to be used throughout the app.